src/components/ResultCard/ResultCard.tsx

```diff
@@ -94,6 +99,12 @@ export default function ResultCard(props: ResultCardOptions) {
         <ResultFavoritesButton item={result} price={salePrice || regularPrice} />
       )}
       {!customClickItemCallback ? resultCardContentWithLink() : resultCardContentWithoutLink()}
+      {selectedOptionsWithAttributes && matchedOptions && !!matchedOptions.length && (
```

Comment:
Can we hide this by default and display it only if the consumer activates it via the config? Something like `resultCardOptions.displayMatchedOptions` (feel free to pick a different name)